src/System.Xml.XPath.XmlDocument/src/System.Xml.XPath.XmlDocument.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>$(NetFrameworkMinimum);netstandard2.0;$(NetMinimum)</TargetFrameworks>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <StrongNameKeyId>Open</StrongNameKeyId>
+    <IsPackable>false</IsPackable>
+    <VersionPrefix>4.3.0</VersionPrefix>
+    <VersionPrefix Condition="'$(IsPackable)' == 'true'">4.4.0</VersionPrefix>
+    <AssemblyVersion>4.0.2.0</AssemblyVersion><!-- NO-INCREMENT: This version is frozen for .NET Standard and .NETCoreApp because the assembly ships inbox. -->
```

Comment:
Can you double check that this one actually ships inbox?  I think we said the reason we needed to ship it was because it was not inbox anywhere.

Reply:
Calling `typeof(System.Xml.XPath.XDocumentExtensions).Assembly.CodeBase` gave me these answers for the following frameworks:

- 2.1 used the nuget package: `file:///C:/Users/calope/source/repos/tools/AssemblyChecker/Downloader/bin/Debug/netcoreapp2.1/win-x86/System.Xml.XPath.XDocument.dll`
- 3.1 used the nuget package: `file:///C:/Users/calope/source/repos/tools/AssemblyChecker/Downloader/bin/Debug/netcoreapp3.1/win-x86/System.Xml.XPath.XDocument.dll`
- 5.0 used the inbox assembly: `file:///C:/Program Files/dotnet/shared/Microsoft.NETCore.App/5.0.17/System.Xml.XPath.XDocument.dll`

I got that AssemblyVersion in line 10 from the 1.1 codebase, from https://github.com/dotnet/corefx/blob/v1.1.13/pkg/Microsoft.Private.PackageBaseline/packageIndex.json#L2637-L2649 
```xml
    "System.Xml.XPath.XmlDocument": {
      "StableVersions": [
        "4.0.1",
        "4.0.0",
        "4.3.0"
      ],
      "BaselineVersion": "4.3.0",
      "AssemblyVersionInPackageVersion": {
        "4.0.1.0": "4.0.1",
        "4.0.2.0": "4.3.0",
        "4.0.0.0": "4.0.0"
      }
    },
```

Reply:
Same number found here: https://github.com/dotnet/corefx/blob/718443a10dbd456b6c8f08ee594a92cfbf2ef8db/src/System.Xml.XPath.XmlDocument/dir.props#L4C1-L4C47

Reply:
Are you sure you're checking the correct type?  Your comments mention System.Xml.XPath.**XDocument**.dll, but this PR is for System.Xml.XPath.**XmlDocument**.dll.  I don't see the latter ever making it inbox, which is why we need to ship it here.

Reply:
I noticed the mistake yesterday and updated the PR description with the right results. I forgot I also had a comment with the wrong info. Let me check this again.

Reply:
I think this one never made it inbox -- if that's the case it's assembly version can be whatever we want (IE: remove from the project file).

Reply:
> Can you double check that this one actually ships inbox? I think we said the reason we needed to ship it was because it was not inbox anywhere.

It does not ship inbox.

- The last time we had the XmlDocument folder was in 1.1: https://github.com/dotnet/corefx/tree/v1.1.13/src/System.Xml.XPath.XmlDocument
- The folder does not exist in 2.1: https://github.com/dotnet/corefx/tree/v2.2.8/src/System.Xml.XPath.XmlDocument

And the updated PR description confirms that we always use the package for the 3 relevant tfms (all 3 have defined types).

So in other words, the versioning properties should not have the assembly version ones, and instead they should just be:

```xml
<VersionPrefix>4.3.0</VersionPrefix>
<VersionPrefix Condition="'$(IsPackable)' == 'true'">4.4.0</VersionPrefix>
<PackageValidationBaselineVersion>4.3.0</PackageValidationBaselineVersion>
```

Reply:
Fixed